sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/Generated/Models/BaseExtraInformation.cs

```diff
@@ -7,7 +7,11 @@
 
 namespace Azure.AI.Language.Conversations
 {
-    /// <summary> The abstract base object for entity extra information. </summary>
+    /// <summary>
+    /// The abstract base object for entity extra information.
+    /// Please note <see cref="BaseExtraInformation"/> is the base class. According to the scenario, a derived class of the base class might need to be assigned here, or this property needs to be casted to one of the possible derived classes.
+    /// The available derived classes include <see cref="EntitySubtype"/>, <see cref="ListKey"/> and <see cref="RegexKey"/>.
+    /// </summary>
     public partial class BaseExtraInformation
```

Comment:
Isn't this supposed to actually be an abstract class as we discussed offline?

Reply:
Its a GA'd type so we can't change it to abstract.

Reply:
Maybe we shouldn't call them "abstract base object" in doc comments at least, then. I'll switch my sign-off status, but I think we should still fix that. It's misleading. Given that, though, why all the extra work to duplicate deserialization in the Unknown* classes?

Reply:
I think the comment still makes sense.  The intent is this is an abstract base class that you should never construct.

Reply:
Offline, you pointed out that first line mentioning "abstract" is from swagger. The generated only added the subsequent 2 lines. In that case, fine. I figured if the C#-specific code generator was doing this, we should probably change it. If it's in swagger, *them* calling it abstract makes sense. It would just be misleading if the C# generator did.